src/system_descriptor.jl

```diff
@@ -87,7 +85,7 @@ function get_supported_args_permutations(v::SystemDescriptor)
             comprehensive_set = union(comprehensive_set, set)
         end
 
-        for values in product(Iterators.repeated(comprehensive_set, length(keys_arr))...)
+        for values in product(repeated(comprehensive_set, length(keys_arr))...)
```

Comment:
Not asking for a change, but be careful about using the word `values` as a variable. It collides with the function of the same name.

Reply:
Will keep it in mind. Thank you.